Remotion/Data/DomainObjects/Infrastructure/Serialization/FlattenedDeserializationInfo.cs

```diff
@@ -60,17 +61,31 @@ public bool GetBoolValue ()
     }
 
     public T GetValue<T> ()
+    {
+      return GetValue<T> (allowNull: false)!;
+    }
+
+    public T? GetNullableValue<T> ()
+    {
+      return GetValue<T> (allowNull: true);
+    }
+
+    [return: MaybeNull]
+    private T GetValue<T> (bool allowNull)
```

Comment:
Whole file: why use the attribute instead of `?`?

Reply:
See ScalarQueryRowAdapter